matrix/room.py

```diff
@@ -57,3 +57,79 @@ async def invite_user(self, user_id: str) -> None:
             )
         except Exception as e:
             raise MatrixError(f"Failed to invite user: {e}")
+
+    async def ban_user(
+        self,
+        room_id: str,
```

Comment:
You don't need to have `room_id` since it's already an attribute of the class. Just use `self.room_id`

Reply:
yey. Idk why I didn't even notice lol